main/python/cmdLineUtils.py

```diff
@@ -18,13 +18,8 @@
 import sys
 from time import sleep
 
-# Support both Python2 and Python3 at the same time
-if sys.version_info.major > 2 :
-    _input = input
-    from itertools import zip_longest
-else:
-    _input = raw_input
-    from itertools import izip_longest as zip_longest
+_input = input
```

Comment:
Also here, similar to the `xrange`, I would just replace `_intput` with `input` in the file